CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+Contributing to RasCAL-
+==========================
+Everyone is welcome to contribute to the RasCAL2 project by either opening an issue (please check that the 
+issue has not been reported already) or submitting a pull request.
+
+Create Developer Environment
+----------------------------
+First begin by creating a fork of the RasCAL-2 repo, then clone the fork
+
+    git clone https://github.com/<username>/RasCAL-2.git
+    cd RasCAL-2
+
+We recommend using anaconda python distribution  for development, a new virtual environment should be 
+created to isolate dependencies. For Windows, first download and install [Microsoft Visual C++](https://aka.ms/vs/16/release/vc_redist.x64.exe). Then run the following
+
+    conda env create -f environment.yaml
+    conda activate rascal2
+
+And finally create a separate branch to begin work
+
+    git checkout -b new-feature
+
+Once complete submit a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) via GitHub. 
+Ensure to rebase your branch to include the latest changes on your branch and resolve possible merge conflicts.
+
+Unit-testing and coverage
+-------------------------
+RasCAL-2 uses the **pytest** module for testing. Proper documentation and unit tests are highly recommended.
```

Comment:
'highly recommended' or 'necessary'? would you merge a contribution that doesn't have these?

Reply:
Depending on the size of the code, if its very clear or difficult to test (as some UI code can be), Yes I would be happy to merge such a contribution. The general idea is that documentation and unit tests are important